src/Raven.Studio/typescript/models/database/index/indexDefinition.ts

```diff
@@ -88,6 +91,16 @@ class indexDefinition {
         this.configuration(this.parseConfiguration(dto.Configuration));
 
         this.additionalSources(_.map(dto.AdditionalSources, (code, name) => additionalSource.create(name, code)));
+
+        this.hasDuplicateFieldsNames = ko.pureComputed(() => {
+            let hasDuplicates = false;
+
+            if (_.uniqBy(this.fields(), field => field.name()).length !== this.fields().length) {
```

Comment:
we can simply return this value, no need to have if, let, etc.

Reply:
ok - thanks - done